### PR TITLE
MINIFICPP-1559 Add new redistributable files to the msi in VS2019 builds

### PR DESCRIPTION
**Pull request by fgerlits**
https://issues.apache.org/jira/browse/MINIFICPP-1559

Visual Studio 2019 x64 contains a new redistributable DLL, `vcruntime140_1.dll`, which is required by VS2019 x64 MiNiFi builds but was not included in the MSI package.

I have added all DLLs in the Visual Studio redistributable directory for all four combinations of VS 2017/2019 and x86/x64.  Some of these DLLs are not currently required by MiNiFi, but they don't take up much disk space (under 100 kB total for the 3 new DLLs in VS2019 x64) and could be useful in the future.

---

Thank you for submitting a contribution to Apache NiFi - MiNiFi C++.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced
     in the commit message?

- [x] Does your PR title start with MINIFICPP-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically main)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE file?
- [ ] If applicable, have you updated the NOTICE file?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check GitHub Actions CI results for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by szaszm** [comment]
I would include all of the redistributables as long as they are not huge.

File: msi/WixWin.wsi [hunk 1]
```diff
@@ -395,6 +371,23 @@ Licensed to the Apache Software Foundation (ASF) under one or more
...
+                <File Id="MiNiFiExe_msvcp140_1" Name="msvcp140_1.dll" KeyPath="no" Source="redist\msvcp140_1.dll"/>
+                <File Id="MiNiFiExe_msvcp140_2" Name="msvcp140_2.dll" KeyPath="no" Source="redist\msvcp140_2.dll"/>
+                <File Id="MiNiFiExe_vccorlib140" Name="vccorlib140.dll" KeyPath="no" Source="redist\vccorlib140.dll"/>
+                <File Id="MiNiFiExe_vcruntime140" Name="vcruntime140.dll" KeyPath="no" Source="redist\vcruntime140.dll"/>
+                <?if $(env.VisualStudioVersion) = 16.0 ?>
+                  <File Id="MiNiFiExe_msvcp140_atomic_wait" Name="msvcp140_atomic_wait.dll" KeyPath="no" Source="redist\msvcp140_atomic_wait.dll"/>
+                  <File Id="MiNiFiExe_msvcp140_codecvt_ids" Name="msvcp140_codecvt_ids.dll" KeyPath="no" Source="redist\msvcp140_codecvt_ids.dll"/>
+                  <?if $(env.build_platform) = x64 ?>
+                    <File Id="MiNiFiExe_vcruntime140_1" Name="vcruntime140_1.dll" KeyPath="no" Source="redist\vcruntime140_1.dll"/>
+                  <?endif ?>
```
Comment (szaszm) [inline msi/WixWin.wsi:387]: Why did you remove vcruntime140_1.dll from 32bit builds? I couldn't find a description of individual DLLs.

Comment (fgerlits) [reply msi/WixWin.wsi:387]: Because my VS2019/x86 Redist folder `C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Redist\MSVC\14.28.29910\x86\Microsoft.VC142.CRT` does not contain it: 
image

I have tested this by building a package and installing it in a blank Windows docker image `mcr.microsoft.com/windows/servercore:1809`; all four combinations of 2017/2019 and x86/x64 worked (meaning the MiNiFi process started up).

**Review by arpadboda — APPROVED** [review]

**Review by szaszm — APPROVED** [review]
